src/common/components/Tooltip/Skin/index.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react';
+import ItemHeader from '../ItemHeader';
+import Background from '../Background';
+import styles from './styles.less';
+
+type Props = {
+  data: Object,
+};
+
+const SkinTooltip = ({ data: { skin } }: Props) => {
+
+  const t = (skin.type === 'Weapon' ? skin.details.type : skin.details.weight_class + ' ' + skin.details.type);
+
+  return (
+    <Background>
+      <ItemHeader name={skin.name} icon={skin.icon} rarity="white" />
+      <span className={styles.skinTypeText}>{t}</span>
```

Comment:
getting close but basically weapon should look like this:
![image](https://user-images.githubusercontent.com/6801309/31612929-3392ed4c-b2cd-11e7-9288-00a2059cb6fe.png)

and armor should look like this:
![image](https://user-images.githubusercontent.com/6801309/31612925-30f1c68a-b2cd-11e7-8422-b0e03b456369.png)